### DISCUSSION TITLE
Is docker install available?

### DISCUSSION THREAD

**Discussion by fahadshery** [danielmiessler/Fabric#818] [Q&A]
Hi,

Just came across this project and wanted to try out. 
I am using whisper, ollama, open-webui, searxng etc. as my `ai stack` using docker. 
It would be great if this project was available using docker for easy deployment with the current `ai stack`?

thanks

**Comment by darylf** [comment]
I'm interested in a Docker container for Fabric as well and can't directly answer your question, but I'd love to know more about your AI stack. If possible, could you share your setup (especially if you have a Docker Compose file)?
